ouroboros-consensus/src/unstable-consensus-testlib/Test/CsjModel/NonEmptySeq.hs

```diff
@@ -0,0 +1,94 @@
+{-# LANGUAGE BangPatterns #-}
```

Comment:
This type doesn't use `AnchoredFragment` for two reasons.

- As a model, I'm trying to keep it as simple as possible. So far, I don't need headers at all, and I can't put points in an `AnchoredFragment` because of the `HasHeader` constraint.

- So far, the model deals with chains that reach all the way back to `Origin`, not _fragments_. As long as that's true, it seems misleading to use `AnchoredFragment`.